Parsers/ASimNetworkSession/Parsers/ASimNetworkSessionSonicWallFirewall.yaml

```diff
@@ -0,0 +1,432 @@
+    , SentPackets = DeviceCustomNumber2Label // DeviceCustomNumberXLabel (cnXLabel=)
+    , ConnectionDuration = DeviceCustomNumber3Label
+    , FirewallAction = fw_action // Firewall Action, such as drop, forward, mgmt, NA
+    , User = susr // Logged-in username associated with the log event.
+    , AppRulePolicyId = af_polid // App Rule Policy ID.
+    , AppRulePolicyName = af_policy // App Rule Policy Name.
+    , AppRuleService = af_service // App Rule Service Name.
+    , AppRuleType = af_type // App Rule Policy Type.
+    , AppRuleObject = af_object // App Rule Object Name.
+    , AppRuleObjectContent = contentObject // App Rule Object Content.
+    , AppRuleAction = af_action // App Rule Action.
+    , SourceIPv6Address = srcV6 // Source IPv6 IP
+    , DestinationIPv6Address = dstV6 // Destination IPv6 IP
+    , ServiceName = appName // Non-signature Application name that matches the Application ID "app" or "f".
+    , AppFullString = appcat // The full "<CATEGORY> <APPNAME> -- <SIGNAME>" string.
+    , AppIDNumber = app // Numeric Application ID. Not the same as "ApplicationProtocol".
+    , AppID = appid // Application ID from App Control
+    , AppCategoryID = catid // Application Category ID
+    , AppSignatureID = sid // Application Signature ID
+    , IPSCategoryName = ipscat // IPS Category Name
+    , AntiSpywareCategory = spycat // Anti-Spyware Category
+    , URLPathName = arg // URL. Represents the URL path name.
+    , FileIdentifier = fileid // File hash or URL
+    , DPIInspectedFlow = dpi // Indicates a flow was inspected by DPI. Applies only to Connection Closed messages.
+    , DestinationPortDPT = dpt // Destination port
+    , NATDestinationPort = dnpt // NAT'ed destination port
+    , SourcePortSPT = spt // Source port
+    , NATSourcePort = snpt // NAT'ed source port
+    , BladeID = bid // Blade ID
+    , AuditID = auditId // Audit ID of the Configuration Auditing event.
+    , AuditTransactionID = tranxId // Transaction ID of the entries pertaining to the same transaction or batch.
+    , AuditUserPrivileges = userMode // User Mode. Displays the privileges assigned to a user.
+    , AuditGroupName = grpName // Labels a group of data usually associated with a table (ex: VPN Policy list, Syslog Server list, etc.)
+    , AuditGroupIndex = grpIndex // Labels the instance of the data group (ex: a VPN policy name, Syslog server entry in the list, etc.)
+    , AuditOldValue = oldValue // The previous value associated with the auditing event.
+    , AuditNewValue = newValue // The new value associated with the auditing event.
+    , AuditTime = auditTime // The timestamp indicates the time configuration was changed.
+    , AuditPath = auditPath // The logical location of the configuration setting.
+    , UUID = uuid
+  | project-away
+      CEFLogMsgSeverity
+    , DeviceEventCategory
+    , gcat
+    , RequestMethod
+    , ipspri
+    , spypri

```

Comment:
there are many fields which are outside of NetworkSession schema. Please remove them.

Reply:
Can you please clarify this item? These fields highlighted in the snippet are used temporarily and I use 'project-away' to remove them. There are many other fields that are outside of the NetworkSession schema that I keep as they provide enrichment of the log messages and parse the fields into relevant fields. For example, there are several that relate to our Deep Packet Inspection services and our security services signatures.

Reply:
I see now that I can use AdditionalFields to include additional details. I'm making the necessary changes to remove anything not in schema.

Reply:
Approving this PR @jaimeesc . But I still feel we might have added more fields in AdditionalField column than what would be populated in logs.